docs/painless/painless-contexts/painless-min-should-match-context.asciidoc

```diff
@@ -25,4 +25,53 @@ results.
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+Imagine that you want to find seats to performances where your favorite
+actors play. You have a list of favorite actors in mind, and you want
+to find performances that have at least certain minimum number of actors
+from your favorite list. `terms_set` query with
```

Comment:
Instead of "where your favorite actors play", maybe "by your favorite actors."?

The second sentence could be pared down to: "You have a list of favorite actors in mind, and you want to find performances where the cast includes at least a certain number of them."